opentreemap/opentreemap/settings/__init__.py

```diff
@@ -36,7 +36,7 @@
 from opentreemap.settings.local_settings import *  # NOQA
 
 INSTALLED_APPS = EXTRA_APPS + INSTALLED_APPS
-MIDDLEWARE_CLASSES += EXTRA_MIDDLEWARE_CLASSES
+MIDDLEWARE = EXTRA_MIDDLEWARE + MIDDLEWARE
```

Comment:
Answered in https://github.com/OpenTreeMap/otm-cloud/pull/389#discussion_r128067599